lib/mix/tasks/openapi.spec.yaml.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Mix.Tasks.Openapi.Spec.Yaml do
+  @moduledoc """
+  Serialize the given OpenApi spec module to a YAML file.
+
+  ## Examples
+
+      $ mix openapi.spec.yaml --spec PhoenixAppWeb.ApiSpec apispec.yaml
+
```

Comment:
All the options should be documented in both tasks.

Reply:
Fixed